docs/sphinx/source/whatsnew/v0.6.1.rst

```diff
@@ -50,6 +50,7 @@ Bug fixes
   and < 1900, fix year 2050 which was returning 0.
 * Fix and improve :func:`~pvlib.solarposition.hour_angle` (:issue:`598`)
 * Fix error in :func:`pvlib.clearsky.detect_clearsky` (:issue:`506`)
+* Fix error in :func:`pvlib.modelchain.ModelChain.infer_spectral_model` (:issue:`507`)
```

Comment:
Issue number is incorrect here, should be #619

Reply:
That's what (I think) I did, fetching the upstream as explained in the github [doc](https://help.github.com/articles/syncing-a-fork/). It's just my fork is old, but clearly the modifications are only a few lines.
When running the test locally, it appears that I have `pytest` v4.0.0, brand new, it may explain the problem.
Otherwise, I have to re-fork the project ?

Reply:
see #623 for existing test failures. this pr will need a new test that would fail without your fix.